config/logging_level.go

```diff
@@ -23,11 +24,12 @@ var globalLogger struct {
 }
 
 // InitLoggingSettings initializes the global logging settings.
-func InitLoggingSettings(logger logging.Logger, cmdLineDebugFlag bool) {
+func InitLoggingSettings(logger, configLogger logging.Logger, cmdLineDebugFlag bool) {
 	globalLogger.mu.Lock()
 	defer globalLogger.mu.Unlock()
 
 	globalLogger.logger = logger
```

Comment:
I'm guessing we're never supposed to write logs to `logger`. But this is the logger we call `SetLevel` on? Can we enhance the name. 

My rule of thumb for naming variables is that -- if only one variable has some type, can just name it after the type. But once two variables have some type, it's easy and effective to come up with non-generic names that describe their purpose and how the two (or more) variables co-exist.